VaticanBAV/et/BAVet141.xml

```diff
@@ -0,0 +1,367 @@
+                                  <dim unit="leaf">10</dim>
+                                  <locus from="79" to="88"/>
+                                </item>
+                                <item xml:id="q11" n="10">
+                                  <dim unit="leaf">10</dim>
+                                  <locus from="89" to="98"/>
+                                </item>
+                                <item xml:id="q12" n="11">
+                                  <dim unit="leaf">10</dim>
+                                  <locus from="99" to="108"/>
+                                </item>
+                                <item xml:id="q13" n="12">
+                                  <dim unit="leaf">10</dim>
+                                  <locus from="109" to="118"/>
+                                </item>
+                                <item xml:id="q14" n="13">
+                                  <dim unit="leaf">10</dim>
+                                  <locus from="119" to="128"/>
+                                </item>
+                                <item xml:id="q15" n="14">
+                                  <dim unit="leaf">10</dim>
+                                  <locus from="129" to="138"/>
+                                </item>
+                                <item xml:id="q16" n="15">
+                                  <dim unit="leaf">10</dim>
+                                  <locus from="139" to="148"/>
+                                </item>
+                                <item xml:id="q17" n="16">
+                                  <dim unit="leaf">8</dim>
+                                  <locus from="149" to="156"/>
+                                </item>
+                              </list>
+                            </collation>
+                          </supportDesc>
+                        
+                        <layoutDesc>
+                          <layout columns="1" writtenLines="23">
+                            <locus from="3r" to="139v"/>
+                          </layout>
+                          <layout columns="2" writtenLines="23">
+                            <locus from="140r" to="155r"/>
+                          </layout>
+                        </layoutDesc>
+                        </objectDesc>
+                      
+                        <handDesc>
+                          <handNote script="Ethiopic" xml:id="h1">
+                            <seg type="ink">Black, red.</seg>
+                            <seg type="rubrication">Titles.</seg>
+                            <date notBefore="1800" notAfter="1899"/>
+                          </handNote>
+                        </handDesc>
+                      
+                    <decoDesc>
+                      <decoNote type="ornamentation" xml:id="d1">
+                        <desc>The Psalms are divided into ten lines, which are marked alternately with red and black dots.</desc>
```

Comment:
I would say, each ten lines of of each psalm is marked by alternatively red of black dots in the margin. The current wording is not clear.